go/api/bitmap_commands_test.go

```diff
@@ -0,0 +1,63 @@
+package api
```

Comment:
Please add licence header to all new files

Reply:
Done